src/app/record/components/top-bar/top-bar.component.ts

```diff
@@ -56,7 +57,7 @@ export class TopBarComponent implements OnInit, OnDestroy {
     if (!this.isPublicRecord) {
       this.getRecord()
     } else {
-      // TODO SUPPORT PUBLIC VIEW
+      this.getPublicRecord()
```

Comment:
I am proposing some improvements here on my last PR https://github.com/ORCID/orcid-angular/pull/759 please review that one before we merge this one.